behavior_metrics/brains/f1/brain_f1_torch.py

```diff
@@ -110,7 +110,7 @@ def execute(self):
             with torch.no_grad():
                 image = self.transformations(img).unsqueeze(0)
                 image = FLOAT(image).to(self.device)
-                prediction = self.net(image).numpy()
+                prediction = self.net(image).cpu().numpy()
```

Comment:
Should this change depend on the type of device used (cuda/cpu)?

Reply:
Thanks for pointing out. It is not required when using only cpu. I have made changes in next commit.